docs/source/interfaces/Indirect Bayes.rst

```diff
@@ -16,6 +16,10 @@ Provides Bayesian analysis routines primarily for use with QENS data.
 Action Buttons
 --------------
 
+Settings
```

Comment:
Make a separate page about the settings GUI, and put links in all the places referring to it.